### PR TITLE
Mldb 256 accuracy multiclass

### PR DESCRIPTION
**Pull request by mailletf**

### CODE REVIEW

**Comment by jeremybarnes** [comment]
@mailletf I added a commit which may help things along.  The thing you were missing was that labels are CellValues, not strings, and when they are passed through column names, they are JSON encoded.

File: container_files/assets/www/doc/builtin/procedures/Accuracy.md [hunk 1]
```diff
@@ -65,15 +75,16 @@ GET http://localhost/v1/procedures/ttnc_test_scorer/runs/1
...
+  "runFinished": "2015-06-05T18:06:19.165Z", 
+  "runStarted": "2015-06-05T18:06:19.119Z", 
   "id": "2015-06-05T18:06:19.118784Z-5bc7042b732cb41f"
 }
 
 ```
 
-The `output` dataset created by this procedure will contain one row per input row, 
+The `output` dataset created by this procedure in `boolean` mode 
+will contain one row per input row,
```
Comment (nicolaskruchten) [inline container_files/assets/www/doc/builtin/procedures/Accuracy.md]: in fact it's one row per score, as we discovered.

File: container_files/assets/www/doc/builtin/procedures/Accuracy.md [hunk 2]
```diff
@@ -83,6 +94,124 @@ with the same row name as in the input, and the following columns:
...
+                "precision": 1.0
+            }
+        },
+        "weighted_statistics": {
+            "f1_score": 0.7200000057220459,
+            "recall": 0.8,
+            "support": 5,
+            "precision": 0.6666666746139527
+        },
+        "confusion_matrix": {
```
Comment (nicolaskruchten) [inline container_files/assets/www/doc/builtin/procedures/Accuracy.md]: hmm we should think about the format here... Perhaps a set of `{actual: "1", predicted: "0", count: 5}` objects in an array?

File: container_files/assets/www/doc/builtin/procedures/Accuracy.md [hunk 3]
```diff
@@ -11,7 +12,16 @@ resulting from the application of a classifier to some input data.
 ## Output
 
 After this procedure has been run, a summary of the accuracy can be obtained via 
-`GET /v1/procedures/<id>/runs/<runid>`, where the `status` field will contain the [Area
+`GET /v1/procedures/<id>/runs/<runid>`. The the `status` field will contain statistics relevant
```
Comment (guyd) [inline container_files/assets/www/doc/builtin/procedures/Accuracy.md]: The the -> the

**Comment by nicolaskruchten** [comment]
+1

**Comment by guyd** [comment]
Beside the comments I made in PR 182, can you please enable the MLDB-1272 test.  I see that you have fixed it.

**Comment by nicolaskruchten** [comment]
@mailletf apologies, but I'm with @guyd on the camelCase/snake_case thing: the output here should be changed to camelCase to match the rest of MLDB's output. Out of curiosity I checked the output of the `classifier.experiment` procedure and there is a mix of camel and snake case there, so we should fix that too. Please LMK if you want me to create a different JIRA issue for that.

**Pull request merged**